### PR TITLE
First pass of converting patchviasocket to python

### PR DESCRIPTION
**Pull request by sverrirab**
This will remove the requirement to install perl on KVM hypervisors

### CODE REVIEW

File: scripts/vm/hypervisor/kvm/patchviasocket.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    msg = MESSAGE.format(key=pub_key, cmdline=cmdline)
+
+    if not os.path.exists(sock_file):
+        print("ERROR: {0} socket not found".format(sock_file))
+        return 1
+
+    try:
+        s = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        s.connect(sock_file)
+        s.sendall(msg)
```
Comment (HrWiggles) [inline scripts/vm/hypervisor/kvm/patchviasocket.py]: Should there not be a call to `s.close()` after `s.sendall(msg)`?

**Comment by HrWiggles** [comment]
Do the new files not need comments specifying license?

**Comment by ProjectMoon** [comment]
To get into Apache they do.

File: scripts/vm/hypervisor/kvm/patchviasocket.py (hunk 1 above)
Comment (sverrirab) [inline scripts/vm/hypervisor/kvm/patchviasocket.py]: Fixed in second commit

**Comment by HrWiggles** [comment]
lgtm

**Pull request merged**